dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/DotTemplateTool.java

```diff
@@ -37,32 +31,14 @@
  * Created by Jonathan Gamba
  * Date: 10/25/12
  */
-public class DotTemplateTool implements ViewTool {
+public class DotTemplateTool  {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)